src/main/java/ddingdong/ddingdongBE/domain/form/service/FacadeUserFormServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package ddingdong.ddingdongBE.domain.form.service;
+
+import ddingdong.ddingdongBE.domain.form.entity.Form;
+import ddingdong.ddingdongBE.domain.form.service.dto.query.FormSectionQuery;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class FacadeUserFormServiceImpl implements FacadeUserFormService {
+
+  private final FormService formService;
+
+  @Override
+  public FormSectionQuery getFormSection(Long formId) {
+    Form form = formService.getById(formId);
+    return FormSectionQuery.of(form.getTitle(), form.getDescription(), form.getSections());
```

Comment:
p4)
사소한거긴 한데 FormSectionQuery.of() 메서드의 파라미터로 넘기는 값이 모두 form 객체의 필드라면 form 객체를 넘겨 팩토리 메서드에서 필드에 접근하는 것이 더 나아보입니다!
```suggestion
    return FormSectionQuery.from(form);
```

Reply:
확실히 그렇네요! 수정했습니다!